site/en/docs/extensions/support-feedback/submit-a-feature-request/index.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: 'layouts/doc-post.njk'
+title: Submit a feature request
+description: 'TBD'
+date: 2023-10-11
+tags:
+  - extensions
+---
+
+## Submit a feature request
+
+If you identify a feature (especially if it's related to an experimental API) that you believe could improve the extension's platform, file a request as described below.
+
+1.  Before submitting a feature request, [search the Chromium issue tracker](/docs/extensions/support-feedback/find-a-bug) to verify that your feature has not already been proposed.
+1.  File an issue in [the issue tracker][https://crbug.com]. Be as explicit as possible when filling out the bug. The easier it is to reproduce the issue, the greater the chance it will be fixed promptly.
```

Comment:
```suggestion
1.  File an issue in [the issue tracker](https://crbug.com). Be as explicit as possible when filling out a feature request. The easier it is to understand the issue, the greater the chance it will be taken into consideration.
```